packages/build/tests/commands/snapshots/tests.js.md

```diff
@@ -21,6 +21,7 @@ Generated by [AVA](https://ava.li).
       featureFlags: {}␊
       repositoryRoot: /file/path␊
       testOpts:␊
+        mockPluginsList: true␊
```

Comment:
Most test snapshots are updated due to `testOpts` being printed in build logs in debug mode.